src/node_report.cc

```diff
@@ -969,8 +981,10 @@ void GetNodeReport(Isolate* isolate,
   if (isolate != nullptr) {
     env = Environment::GetCurrent(isolate);
   }
+  bool exclude_network
+    = per_process::cli_options->per_isolate->per_env->report_exclude_network;
```

Comment:
This should only be the fallback when env is not available. Otherwise API settings would not affect this.

Reply:
fixed :)